README.adoc

```diff
@@ -40,6 +40,9 @@ It creates the following resources:
 * An optional internet gateway
 * An optional NAT gateway
 * An optional service gateway
+* An optional Dynamic Routing Gateway
```

Comment:
The other items in the list are lowercase, so dynamic routing gateway should be too.